cmd/csaf_provider/create.go

```diff
@@ -87,20 +90,85 @@ func createFeedFolders(c *config, wellknown string) error {
 // and writes the CSAF field inside the file.
```

Comment:
Please also correct the comment to reflect the new functionality.

Reply:
[x] done